data/korath/korath outfits.txt

```diff
@@ -178,6 +204,236 @@ outfit "Control Transceiver"
 	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."
 
 
+outfit "Afterburner (Asteroid Class)"
+	plural "Afterburners (Asteroid Class)"
+	category "Engines"
+	"cost" 136000
+	thumbnail "outfit/tiny korath afterburner"
+	"mass" 10
+	"outfit space" -10
+	"engine capacity" -10
+	"fuel capacity" 100
+	"ramscoop" 0.5
```

Comment:
This makes it a better fuel carrier than the independent fuel pod, with the added benefit of some afterburner in exchange for solely engine space. With the ramscoop included it's a not much much smaller but much less efficient fuel processor with the afterburner benefits.

Removing ramscoop would also suggest a remodel, since that includes parts directly appealing to the other Exile outfits with that tech.

A volcano is stronger afterburner, yes, but it would eat up (at least) 8 more of a fighter's outfit space.

Reply:
RIP the World-Ship Chaser if you remove the ramscoop, which is the only "fighter" that's actually been using this so far.  In some variants, I've been throwing this on specifically for the ramscoop rather than the thrust, when there's not enough space for a Fuel Processor.

How about keeping the ramscoop, but reducing the thrust, or possibly, carefully, increasing the energy, heat, or fuel consumption?

Reply:
Alright,  I see the changes you made, and I think it looks good. I'd still like to suggest dropping the space and mass, though. Even if it is just down to 9, I'd like to see just a little more density than it currently does. And have it be a *differently* shaped piece in the outfitting game, rather than be identically sized to the volcano.

Reply:
Nine is fine.  I doubt the similarity was intentional; at the time of its conception, there was no Volcano afterburner, just the plain Afterburner which was IIRC 15 tonnes.